eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -464,6 +466,22 @@ object Commitments {
 
   def revocationHash(seed: ByteVector32, index: Long): ByteVector32 = Crypto.sha256(revocationPreimage(seed, index))
 
+  /** /!\ WARNING : never remove a feerate from the list, because we will only try thosecan only add more */
+  val customRemoteSigsFeerates: Seq[Long] = List(1 sat, 2 sat, 5 sat, 10 sat).map(satPerByte => feerateByte2Kw(satPerByte.toLong))
+
+  def computeCustomRemoteSigs(commitments: Commitments, keyManager: KeyManager, remoteCommitIndex: Long, spec: CommitmentSpec, remotePerCommitmentPoint: PublicKey): Seq[CustomRemoteSig] = {
+    import commitments._
+    if (spec.htlcs.isEmpty) { // TODO: we could use htlcTxs.isEmpty to ignore trimmed htlcs
```

Comment:
I think we should stick to `spec.htlcs.isEmpty` which is simpler, otherwise there are multiple places where we'd need to match the logic and it could become non-trivial (in `remoteSpentOther` for example).

Reply:
Especially if some trimmed htlcs become non-trimmed due to lower feerates 😓

Reply:
Definitely, even though what I think we would do is to consider htlcs trimmed at the *current* feerate, regardless of what *custom* feerate we sign (this way we're sure we're always signing a commit tx without htlcs at all feerates). But that is more subtle so it's more risky!